src/main/java/frc/robot/subsystem/DriveSubsystem.java

```diff
@@ -1,27 +1,65 @@
 package frc.robot.subsystem;
 
 import com.ctre.phoenix6.hardware.TalonFX;
+<<<<<<< HEAD
+import com.kauailabs.navx.frc.AHRS;
+import com.momentum4999.motune.PIDTuner;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.filter.LinearFilter;
+=======
 import com.pathplanner.lib.controllers.PPHolonomicDriveController;
 import com.pathplanner.lib.util.DriveFeedforwards;
 import com.revrobotics.spark.SparkLowLevel.MotorType;
 import com.revrobotics.spark.SparkMax;
+>>>>>>> 5e13cd9fa02c26f9ef30f4835df78748d244b002
```

Comment:
These are merge conflicts. Git adds the `<<<<`, `====`, and `>>>>` to indicate what came from the left side and the right side of the merge. When resolving merge conflicts, it's your job to figure out which side you want to keep (or both), then edit the file so that you have only what you want to keep. You shouldn't leave the `<<<<`, `====`, or `>>>>` as they're not correct java syntax and will cause the code not to compile.